readme.md

```diff
@@ -27,8 +27,8 @@ The page should match the design Pixel Perfect: all the sizes, colors and distan
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+- [DEMO LINK](https://github.com/IlyaPerfomance/layout_moyo-header)
+- [TEST REPORT LINK](https://github.com/IlyaPerfomance/layout_moyo-header/report/html_report/)
```

Comment:
Fix the demo and test report links and update it in the PR description 
<img width="522" alt="image" src="https://github.com/user-attachments/assets/6734b675-e4e7-470e-b527-2ad1c5eaaf25">
```suggestion
- [DEMO LINK](https://IlyaPerfomance.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://IlyaPerfomance.github.io/layout_moyo-header/report/html_report/)
```